src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  margin: 0;
```

Comment:
DRY, you've repeated `margin: 0 `3 times, let's combine this rule for all 3 elements like this
```
.paragraph,
.title,
.etc {
    margin: 0
}
```